poker-hand-evaluator/src/evaluator/CardTable.java

```diff
@@ -0,0 +1,35 @@
+package evaluator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CardTable {
```

Comment:
right now it seems to be implied that there is only one table throughout the application - how would you have to change it in a way that your application supports having `n>1` tables?

Reply:
also, if everything is static, always remember it is a good design decision to put a private constructor there - you don't need any instances